examples/files.js

```diff
@@ -107,6 +107,7 @@ var files = {
 		"webgl_loader_nodes",
 		"webgl_loader_obj",
 		"webgl_loader_obj_mtl",
+		"webgl_loader_obj_obj2_verify",
```

Comment:
How about `webgl_loader_obj2_verify`?

Reply:
It uses both `OBJLoader` and `OBJLoader2` that's why I chose that name. You decide!